### ISSUE TITLE
REMINDER: Linux Compiled Bugfix

### ISSUE THREAD

**Issue by OcelotWalrus** [Dungeons-of-Kathallion/Bane-Of-Wargs#19]
**Describe the bug**
When running the automatic compiled game in the Continuous Build release for linux, everything work fine except the downloading game data process. This should be fixed

**To Reproduce**
1. Go to 'https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/releases/tag/9.9.9-continuous'
2. Click on 'Bane-Of-Wargs'
3. Scroll down to 'run `$ chmod +x Downloads/Bane-Of-Wargs && ./Downloads/Bane-Of-Wargs`'
4. See errors

**Expected behavior**
Game data should be able to be downloaded at the game start

**Log File**
```logs
2024-01-07 09:35:25,332 INFO: Checking program directory
2024-01-07 09:35:25,332 INFO: Downloading game data to update it
2024-01-07 09:35:25,333 INFO: Downloading game yaml schemas files from github
2024-01-07 09:35:25,335 WARNING: An error occurred when downloading game data to '/home/lolo/.config/Bane-Of-Wargs/game/schemas'.
2024-01-07 09:35:25,335 DEBUG: Install the requests package to use the github FS
2024-01-07 09:35:25,836 INFO: Downloading game data files from github
2024-01-07 09:35:25,838 WARNING: An error occurred when downloading game data to '/home/lolo/.config/Bane-Of-Wargs/game/data'.
2024-01-07 09:35:25,839 DEBUG: Install the requests package to use the github FS
2024-01-07 09:35:26,339 INFO: Downloading game images .txt files from github
2024-01-07 09:35:26,341 WARNING: An error occurred when downloading game data to '/home/lolo/.config/Bane-Of-Wargs/game/imgs'.
2024-01-07 09:35:26,342 DEBUG: Install the requests package to use the github FS
2024-01-07 09:35:26,843 INFO: Process of downloading game data to update it completed
2024-01-07 09:35:26,843 INFO: Opening player '/home/lolo/.config/Bane-Of-Wargs/preferences.yaml'
2024-01-07 09:35:26,852 INFO: Validating file '/home/lolo/.config/Bane-Of-Wargs/preferences.yaml' data: '[({'auto update': False, 'latest preset': {'plugin': 'none', 'save': 'Cromha MCcloud', 'type': 'vanilla'}, 'speed up': True, 'theme': 'greenblue', 'title style': 1}, '/home/lolo/.config/Bane-Of-Wargs/preferences.yaml')]' with schema '/home/lolo/.config/Bane-Of-Wargs/game/schemas/preferences.yaml'
2024-01-07 09:35:27,359 INFO: Printing title
2024-01-07 09:35:29,292 INFO: Printing title
```

**Comment by OcelotWalrus** [comment]
Tried to add `requests` module to `requirements.txt`...

**Comment by OcelotWalrus** [comment]
> Tried to add `requests` module to `requirements.txt`...

Bug fixed by PR #20 !
